src/main/java/ch/uzh/ifi/hase/soprafs22/controller/LobbyController.java

```diff
@@ -127,4 +131,22 @@ public List<LobbyGetDTO> getLobby() {
         return lobbiesGetDTOs;
 
     }
+
+    @PostMapping("{API_VERSION}/game/lobby/{id}/player")
```

Comment:
Sorry, I changed the "API_VERSION" variable in the newest master to an "apiVersion". It won't work with API_VERSION anymore.